web/core/components/issues/issue-layouts/filters/header/filters/filters-selection.tsx

```
@@ -242,6 +244,31 @@ export const FilterSelection: React.FC<Props> = observer((props) => {
             />
           </div>
         )}
+
+          {ISSUE_ADDITIONAL_PROPERTIES.map((prop: any) => (
+            <div className="py-2">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)